source/Octopus.Tentacle.Tests.Integration/Support/TentacleVersions.cs

```diff
@@ -20,11 +20,11 @@ public static class TentacleVersions
         public static Version v6_3_451_NoCapabilitiesService = new("6.3.451");
 
         // Last version of v7 with ScriptServiceV2
-        public static readonly Version v7_1_189_SyncHalibutAndScriptServiceV2 = new("7.1.189");
+        public static readonly Version v7_1_189_ScriptServiceV2Added = new("7.1.189");
```

Comment:
Could we keep the new names?

Reply:
Yeah, I just did a pure revert. Let me re-add those names